test/runtests.jl

```diff
@@ -22,7 +22,7 @@ testnames = [n[6:(end-3)] for n in alltests]
 
 to_test = alltests # uncomment this line to run all tests or add individual files below
 [
-# "test-create-landmask.jl",
+# "test-latlon.jl",
```

Comment:
this looks like an unexpected change – the code is nothing to do with latlon or create-landmask